### DISCUSSION TITLE
[1304930] The Outlast Trials

### DISCUSSION THREAD

**Discussion by owwkmidream** [pjy612/SteamManifestCache#422] [Share]
appid:
```
1304930
```
keys:
```
1304931;81611074b6fd0d2e108d34562aca73115d01d6bea3f08092b8d98aad42c2f0fb
```

t.me/AnneHappiy

[1304931_811045739035008155.zip](https://github.com/pjy612/SteamManifestCache/files/14052792/1304931_811045739035008155.zip)

**Comment by owwkmidream** [comment]
@pjy612 请问tools的lua有api文档吗

**Reply by pjy612** [reply]
没有文档，只有以前 beta1 的 模板。
其他的可以看 tg 群 其他管理分享的 插件生成工具。
从生成后的 lua 里 看可用函数。
